pkg/endpoint/endpoint.go

```diff
@@ -315,10 +403,10 @@ func OptionChanged(key string, value bool, data interface{}) {
 	}
 }
 
-func (e *Endpoint) ApplyOpts(opts option.OptionMap) bool {
+func (e *Endpoint) ApplyOpts(opts map[string]string) bool {
 	// We want to be notified if the packets are dropped with the learn traffic
-	if val, ok := opts[OptionLearnTraffic]; ok && val {
-		opts[OptionDropNotify] = true
+	if val, ok := opts[OptionLearnTraffic]; ok && val == "enabled" {
```

Comment:
Just to be safe, you're already changed all "true" to "enabled" before calling this function, right?